mvnw

```diff
@@ -0,0 +1,259 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
```

Comment:
Do we use mvnw? I thought it was an older wrapper script we no longer use.

Reply:
tinkar-core has these in every maven module. I'm not a fan but was trying to be consistent.